### ISSUE TITLE
💡 Read additional files from QGIS project file

### ISSUE THREAD

**Issue by fgravin** [qgis/qgis-js#48]
This is a proposal to slightly improve the developer experience while using the qgis-js API.

At the moment, we have to list all the files that are used within the QGIS project, beside the QGIS project file itself.
eg.
```js
const source =
  "/Glacier_3857";
const files = ["glaciers.qgs","glaciers.gpkg", "glaciers_attachments.zip"];
```

It would be great if the additional files could be guessed by the API by reading the QGIS project file.

**Comment by boardend** [comment]
Agree, it would be nice to just point to a project and query all involved files from there

Would be easy to expose/implement, if QGIS provides something like this internally, cc @wonder-sk.

Note that there is also the [`@qgis-js/utils`](https://github.com/qgis/qgis-js/blob/main/packages/qgis-js-utils/README.md) package that helps to download many files from a web server, a GitHub repo, or the local file system
